### DISCUSSION TITLE
Welcome to windows-terminals Discussions!

### DISCUSSION THREAD

**Discussion by rjcarneiro** [rjcarneiro/windows-terminals#26] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Linkeb3** [comment]
<img width="1402" height="1122" alt="defPM" src="https://github.com/user-attachments/assets/5b9019be-35f6-4f27-acde-ddb1ed132d6b" />
[old-parchment-terminal-theme.md](https://github.com/user-attachments/files/31121537/old-parchment-terminal-theme.md)
<img width="1536" height="1024" alt="parchment" src="https://github.com/user-attachments/assets/0b9a2a90-ee5a-4717-9db4-50a2e103c9ba" />

**Reply by Linkeb3** [reply]
# Old Parchment

A Windows Terminal theme inspired by aged parchment, antique books, and manuscript-style typography.

This setup combines:

- A warm parchment color palette
- Optional parchment background image
- Windows Terminal profile settings
- A matching Oh My Posh prompt
- Compatibility with WSL, PowerShell, Command Prompt, and terminal-based tools such as Claude Code

## Preview

The theme is designed around dark brown text, muted burgundy and olive accents, and a warm parchment background.

```text
❧ user@machine : ~/projects/example  ❦ main

❯
```

---

## Windows Terminal Color Scheme

Add the following object to the `"schemes"` array in Windows Terminal's `settings.json`:

```json
{
    "name": "Old Parchment",
    "background": "#D8B878",
    "foreground": "#2B1B0E",

    "black": "#2B1B0E",
    "red": "#7A271A",
    "green": "#4D5A2B",
    "yellow": "#8A5A20",
    "blue": "#455568",
    "purple": "#65405C",
    "cyan": "#3F625D",
    "white": "#CBB07A",

    "brightBlack": "#66513A",
    "brightRed": "#9A3928",
    "brightGreen": "#65733D",
    "brightYellow": "#A8752B",
    "brightBlue": "#596E82",
    "brightPurple": "#825774",
    "brightCyan": "#527C73",
    "brightWhite": "#F0D99D",

    "cursorColor": "#3A210F",
    "selectionBackground": "#B89558"
}
```

---

## Apply the Theme to a Profile

Add:

```json
"colorScheme": "Old Parchment"
```

to any Windows Terminal profile.

Example:

```json
{
    "guid": "{PROFILE-GUID}",
    "hidden": false,
    "name": "Ubuntu",
    "source": "Windows.Terminal.Wsl",
    "colorScheme": "Old Parchment",
    "font": {
        "face": "Cascadia Mono",
        "size": 11
    },
    "opacity": 100,
    "useAcrylic": false
}
```

The same color scheme can also be used with PowerShell, Command Prompt, SSH profiles, or other Windows Terminal shells.

---

## Optional Parchment Background

For the full parchment effect, use a parchment texture as the terminal background.

Example:

```json
"backgroundImage": "C:\\path\\to\\parchment-terminal.png",
"backgroundImageOpacity": 0.18,
"backgroundImageStretchMode": "uniformToFill",
"opacity": 100,
"useAcrylic": false
```

A value between approximately `0.12` and `0.25` works well for `backgroundImageOpacity`.

Lower values produce a cleaner coding environment, while higher values make the parchment texture more visible.

Example:

```json
{
    "colorScheme": "Old Parchment",
    "backgroundImage": "C:\\path\\to\\parchment-terminal.png",
    "backgroundImageOpacity": 0.18,
    "backgroundImageStretchMode": "uniformToFill",
    "opacity": 100,
    "useAcrylic": false
}
```

---

## Matching Oh My Posh Theme

The following Oh My Posh configuration uses manuscript-inspired symbols and colors that complement the Windows Terminal palette.

Save it as:

```text
~/.config/oh-my-posh/parchment.omp.json
```

```json
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "version": 3,
  "final_space": true,
  "blocks": [
    {
      "type": "prompt",
      "alignment": "left",
      "segments": [
        {
          "type": "text",
          "style": "plain",
          "foreground": "#7A271A",
          "template": "❧ "
        },
        {
          "type": "session",
          "style": "plain",
          "foreground": "#59351D",
          "template": "{{ .UserName }}@{{ .HostName }}"
        },
        {
          "type": "text",
          "style": "plain",
          "foreground": "#8A5A20",
          "template": " : "
        },
        {
          "type": "path",
          "style": "plain",
          "foreground": "#3E4D2A",
          "properties": {
            "style": "folder"
          },
          "template": "{{ .Path }}"
        },
        {
          "type": "git",
          "style": "plain",
          "foreground": "#65405C",
          "properties": {
            "branch_icon": "⑂ ",
            "fetch_status": true
          },
          "template": "  ❦ {{ .HEAD }}{{ if .Working.Changed }} <#9A3928>{{ .Working.String }}</>{{ end }}"
        },
        {
          "type": "text",
          "style": "plain",
          "foreground": "#7A271A",
          "template": "\n❯ "
        }
      ]
    }
  ]
}
```

### Bash

```bash
eval "$(oh-my-posh init bash --config ~/.config/oh-my-posh/parchment.omp.json)"
```

### Zsh

```bash
eval "$(oh-my-posh init zsh --config ~/.config/oh-my-posh/parchment.omp.json)"
```

---

## Font Recommendations

The theme works best with a readable monospace font.

Recommended options:

- Cascadia Mono
- Cascadia Code
- JetBrains Mono
- IBM Plex Mono
- Iosevka
- Courier Prime

Decorative manuscript fonts are generally not recommended as the primary terminal font because they may lack programming glyphs or produce poor character alignment.

---

## Suggested Adjustments

### Cleaner appearance

```json
"backgroundImageOpacity": 0.14
```

### Stronger parchment texture

```json
"backgroundImageOpacity": 0.22
```

### Darker foreground

```json
"foreground": "#24160B"
```

### Warmer base background

```json
"background": "#CFA96A"
```

---

## Example File Layout

```text
windows-terminal/
├── parchment-terminal.png
└── parchment.omp.json
```

The background image is optional. Without it, the `Old Parchment` color scheme still provides a usable warm, paper-like terminal palette.

---

## Notes

Windows Terminal handles the color palette, font, and optional background image.

Oh My Posh is optional and only changes the shell prompt.

Applications running inside the terminal inherit the terminal's ANSI colors and background, so the theme works well with command-line development tools, SSH sessions, Git, and terminal-based AI assistants.
